docscan/session/create/check/document_authenticity_test.go

```diff
@@ -0,0 +1,21 @@
+package check
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+func ExampleRequestedDocumentAuthenticityCheckBuilder() {
+	check, err := NewRequestedDocumentAuthenticityCheckBuilder().Build()
+
+	if err != nil {
+		fmt.Printf("error: %s", err.Error())
+		return
+	}
+
+	data, _ := json.Marshal(check)
+	fmt.Println(string(data))
+	// Output: {"type":"ID_DOCUMENT_AUTHENTICITY","config":{}}
+
+	// TODO: check we want config={}?
```

Comment:
I think we can remove this comment - we confirmed that omitting was more desirable, but empty `{}` was also fine